extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -1,4 +1,4 @@
-/* global UserLoginModal, wgCanonicalSpecialPageName, wgMainPageTitle, wgArticlePath, wgScriptPath, wgUserLanguage */
+/* global UserLoginModal, wgScriptPath, wgUserLanguage */
```

Comment:
Note: the name of this file changes with UC-200. We'll probably merge this PR first, but just in case we don't, heads up, there may be some conflicts to sort out.